features/bevragen/persoon-beperkt/verblijfplaats/adres/fields.feature

```diff
@@ -0,0 +1,34 @@
+# language: nl
+
+Functionaliteit: Persoon beperkt: verblijfplaats binnenland (adres) velden vragen met fields
+
+  Abstract Scenario: 'functie adres (10.10)' wordt gevraagd met field pad '<fields>'
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens
+    | naam                  | waarde   |
+    | geslachtsnaam (02.40) | Maassen  |
+    | geboortedatum (03.10) | 19830526 |
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | naam                  | waarde |
+    | functie adres (10.10) | W      |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam                 | waarde |
+    | gemeentecode (92.10) | 0518   |
+    | straatnaam (11.10)   | Spui   |
+    Als personen wordt gezocht met de volgende parameters
+    | naam          | waarde                              |
+    | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam | Maassen                             |
+    | geboortedatum | 1983-05-26                          |
+    | fields        | <fields>                            |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
```

Comment:
In het geval van het eerste voorbeeld moet hier ook het veld 'verblijfadres/straat' worden teruggegeven.

Reply:
Nee, dat is niet zo. AdresBeperkt heeft maar één veld: functieAdres. Dus fields=verblijfplaats levert hetzelfde als fields=verblijfplaats.functieAdres